<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>4</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDockWidget" name="dockWidget_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>93</height>
         </size>
        </property>
        <widget class="QWidget" name="dockWidgetContents_5">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QTreeView" name="treeFileExplorer">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1157</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNowy">
    <property name="title">
     <string>Nowy</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdytuj">
    <property name="title">
     <string>Edytuj</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuZapisz">
    <property name="title">
     <string>Zapisz</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuUstawienia">
    <property name="title">
     <string>Ustawienia</string>
    </property>
   </widget>
   <addaction name="menuNowy"/>
   <addaction name="menuEdytuj"/>
   <addaction name="menuZapisz"/>
   <addaction name="menuUstawienia"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUstawienia">
   <property name="text">
    <string>Ustawienia</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
